src/django/api/views/facility/facility_list_view_set.py

```diff
@@ -62,7 +63,7 @@
     FacilityListItemsQueryParamsSerializer,
 )
 from api.extended_fields import (
-    create_extendedfields_for_listitem
+    create_extendedfields_for_single_item
```

Comment:
**Non blocking question:**
Why you replace the method to single item as it list upload?

Reply:
It won't work with `create_extendedfields_for_listitem` since the data from the ContriCleaner has already been adapted for `create_extendedfields_for_single_item`, and another advantage is that the ContriCleaner did the same thing as `create_extendedfields_for_listitem` in terms of assigning `processing_type` and `facility_type`.